app/main/views/g_cloud.py

```diff
@@ -29,7 +28,42 @@
 
 @main.route('/g-cloud')
 def index_g_cloud():
-    return render_template('index-g-cloud.html')
+    # if there are multiple live g-cloud frameworks, assume they all have the same lots
+    all_frameworks = data_api_client.find_frameworks().get('frameworks')
+    framework = framework_helpers.get_latest_live_framework(all_frameworks, 'g-cloud')
+
+    lot_browse_list_items = list()
+    for lot in framework['lots']:
+        lot_item = {
+            "link": url_for('.search_services', lot=lot['slug']),
+            "title": lot['name']
+        }
+
+        # TODO proper lot body/subtext for G9 - G7/G8 content moved here temporarily from template
```

Comment:
It's this ticket: https://trello.com/c/qF5KbUwX/377-update-content-on-homepage-for-g-cloud

Although right now there are discussions about dropping that page entirely, and having links direct from the DM home page. Not sure if that will come to anything before go-live though.